### ISSUE TITLE
How do I change the fade-in animation for the header and list?

### ISSUE THREAD

**Issue by bpeck81** [zyslife/react-native-head-tab-view#143]
## Describe the bug
When the page loads the header and list fade-in with an animation. I don't want this style. How can I change this to just have these components appear rather than fade in?

### Expected behavior
The header and flatlist don't have an animation and just appear.

## Package versions  

- React: "18.2.0"
- React Native: "0.71.6"
- React-Native-Gesture-Handler "2.9.0"

**Comment by kormocicsaba** [comment]
Hey! Did you find a solution?

**Comment by bpeck81** [comment]
No, I went with the pedrobern library. It’s not perfect, but I got it working well enough for my use case. I also figured out you can use a tab view with a scroll view and sticky header prop, it just doesn’t let you swipe between tabs.

**Comment by bpeck81** [comment]
No, I went with the pedrobern library. It’s not perfect, but I got it working well enough for my use case. I also figured out you can use a tab view with a scroll view and sticky header prop, it just doesn’t let you swipe between tabs.
